commands/transform.ts

```diff
@@ -4,11 +4,7 @@ import { run as jscodeshift } from 'jscodeshift/src/Runner'
 import { bold } from 'picocolors'
 import prompts from 'prompts'
 import { TRANSFORM_OPTIONS } from '../config'
-
-export function onCancel() {
-  console.info('> Cancelled process. Program will stop now without any actions. \n')
-  process.exit(1)
-}
+import { onCancel, promptSource } from '../utils/share'
```

Comment:
this `share` filename needs adjustment. It does not tell what can be found inside. From what I can see you collect there utility methods for commands

Reply:
Yeah, I gave it that name without thinking too much, I just wanted to separate it